// Copyright 2019 Zeppelin Bend Pty Ltd
// This file is part of cimbend.
// 
// cimbend is free software: you can redistribute it and/or modify
// it under the terms of the GNU Affero General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
// 
// cimbend is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU Affero General Public License for more details.
// 
// You should have received a copy of the GNU Affero General Public License
// along with cimbend.  If not, see <https://www.gnu.org/licenses/>.


syntax = "proto3";
option java_multiple_files = true;
option java_package = "com.zepben.protobuf.cim.iec61970";

package zepben.cim.iec61970.phase;

/*
* Phases. Note UNKNOWN_PHASE is implicitly also the default!
*/
enum PhaseCode {
    UNKNOWN = 0;
    ABCN = 1;
    ABC = 2;
    ABN = 3;
    ACN = 4;
    BCN = 5;
    AB = 6;
    AC = 7;
    BC = 8;
    AN = 9;
    BN = 10;
    CN = 11;
    A = 12;
    B = 13;
    C = 14;
    N = 15;
    s1N = 16;
    s2N = 17;
    s12N = 18;
    s1 = 19;
    s2 = 20;
    s12 = 21;
    none = 22;
    X = 23;
    XY = 24;
    XN = 25;
    XYN = 26;
}
